File: sem3/map/Lab/3/toylanguage/src/main/java/com/company/Views/Commands/ExitCommand.java
package com.company.Views.Commands;

import com.company.Models.Expression;
import com.company.Views.Command;

public class ExitCommand extends Command {

    public ExitCommand(String name, String description) {
        super(name, description);
    }

    @Override
    public void execute() {
        System.exit(0);
    }
}

File: sem3/map/~craciunf/c-sharp-toy-language-interpreter/view/ExitCommand.cs
/* package view; */
using System;

namespace ToyLanguageInterpreter {
    public class ExitCommand : Command {
        public ExitCommand(String key, String description):base(key, description) {}

        /* @Override */
        public override void execute() {
            Environment.Exit(0);
        }
    }
}
